drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -112,3 +112,28 @@ matterGeneric:
       - id: 0x0107 # Occupancy Sensor
       - id: 0x0015 # Contact Sensor
     deviceProfileName: motion-contact-battery
+  - id: "matter/air-quality-sensor"
+    deviceLabel: Matter Air Quality Sensor
+    deviceTypes:
+      - id: 0x002C
+    deviceProfileName: air-quality-sensor
+  - id: "matter/air-quality-tuo"
```

Comment:
Drop the vendor name from these generic fingerprints and profiles

Reply:
Done in [39cdb6f](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1422/commits/39cdb6f5aa61b8bcd2614de83c774e438afa8a3d)!